packages/addons/addon-depends/jre-depends/jre-libbluray/package.mk

```diff
@@ -0,0 +1,42 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
+# Copyright (C) 2019-present vpeter
+
+. $(get_pkg_directory libbluray)/package.mk
+
+PKG_NAME="jre-libbluray"
+PKG_DEPENDS_TARGET+=" jdk-x86_64-zulu apache-ant:host"
```

Comment:
How is `jdk-x86_64-zulu` going to build as the target for non-x86_64 architectures?

Reply:
OK, I suppose it probably doesn't matter in this instance, but this should probably be `jdk-x86_64-zulu:host`

Reply:
Because the package is only unpacked it really doesn't matter. But will change everywhere to :host.